code-reviews/README.md

```diff
@@ -77,76 +74,82 @@ Code reviews should use the below guidance and checklists to ensure positive and
     * Does the PR description make sense?
     * Do all the changes logically fit in this PR, or are there unrelated changes?
     * If necessary, are the changes made reflected in updates to the README or other docs? If the changes affect how the user builds code especially.
-
-2. User Facing Changes
+1. User Facing Changes
     * If the code involves a user-facing change, is there a GIF/photo that explains the functionality? If not, it might be key to validate the PR to ensure the change does what is expected.
-    * Ensure UI changes look good without unexpected behavior.  
-
-3. Design  
+    * Ensure UI changes look good without unexpected behavior.
+1. Design
     * Do the interactions of the various pieces of code in the PR make sense?
     * Does the code recognize and incorporate architectures and coding patterns?
 
 ## Code Quality Pass
 
 1. Complexity
-    * Are functions too complex?  
+    * Are functions too complex?
     * Should a function be broken into multiple functions?
     * If a method has greater than 3 arguments, potentially overly complex.
```

Comment:
than 3 arguments, potentially overly complex. -> than 3 arguments, it is potentially...

Reply:
Fixed